pyproject.toml

```
@@ -24,12 +24,13 @@ keywords = [
 ]
 license = { text = "Apache 2.0" }
 classifiers = [
-    'Development Status :: 3 - Alpha',
+    'Development Status :: 4 - Beta',
```

```suggestion
    'Development Status :: 3 - Alpha',
```

Let's keep alpha status until we get feedback from at least a couple of users

---

ok!